.github/workflows/build-and-upload-staging.yml

```diff
@@ -1,70 +1,27 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-name: Build And Upload Staging Artifact
-env:
-  # Use terraform assume role for uploading to ecr
-  AWS_ASSUME_ROLE: ${{ secrets.TERRAFORM_AWS_ASSUME_ROLE }}
-  ECR_OPERATOR_STAGING_IMAGE: ${{ secrets.ECR_OPERATOR_STAGING_IMAGE }}
-
+name: Build and Upload Integration Test Artifact
```

Comment:
probably should rename the file to reflect the name change

Reply:
done